index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>hello world</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1 class="hello">hello world</h1>
+    <p>
+      hello world is the first project of microvese that should be created at
+      second day of microverse program wiht all profesional dacumentation.
+    </p>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate cum
+      sapiente maiores possimus inventore, facilis repellendus dicta fugiat
+      incidunt sed est commodi quam ratione libero laborum modi dignissimos.
+      Officia, porro!
+    </p>

```

Comment:
- [ ] `optional`
Great job going an extra mile and adding more initiative however this was not part of the requirement but I suggest you stick to the requirement for professionalism.